Update PULL_REQUEST_TEMPLATE.md (#1046)
integr8ly/installation: .github/PULL_REQUEST_TEMPLATE.md

<!-- Add any additional information needed. Such as the Jira or GH issue this PR relates to or any other context you feel is necessary.) -->

## Verification Steps
As the verifier of the PR the following process should be done:

> - Ensure the developer of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.

Reviewer: ```suggestion
### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
```

Reviewer: Can we provide steps somewhere else on how to write the installation run output to a file

Author: well it is just logged to std out so ```> tofile.txt ``` is all that is needed

> - Ask the developer do they have a cluster on which you can verify the fresh installation is correct.

Reviewer: ```suggestion
- Verify the fresh installation is correct on cluster provided by PR author 
```

> - Ask the developer to run the upgrade and attach an upgrade log. Ensure and that it exited as expected. If possible look at the tasks that ran and see they match the PR

Reviewer 2: @maleck13 If i'm understanding this right, we're saying he verification no longer requires the verifier to try out the code changes. It is enough for the PR creater to produce proof that it was run?

Author: yes then the verifier should inspect the cluster to ensure it ended in the correct state

> - Ask the developer to run the upgrade and attach an upgrade log. Ensure and that it exited as expected. If possible look at the tasks that ran and see they match the PR

Reviewer: ```suggestion
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 
```

<!-- Add any additional information needed. Such as the Jira or GH issue this PR relates to or any other context you feel is necessary.) -->

## Verification Steps
As the verifier of the PR the following process should be done:

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

> - Re check the cluster and ensure the changes expected happend after the upgrade.

Reviewer: Only need one of these lines.

Other than that, the PR is good to go.
